code/game/machinery/doors/poddoor.dm

```diff
@@ -53,11 +53,38 @@
 	if(!hasPower())
 		open()
 
-/obj/machinery/door/poddoor/multi_tile // Whoever wrote the old code for multi-tile spesspod doors needs to burn in hell.
+ // Whoever wrote the old code for multi-tile spesspod doors needs to burn in hell. - Unknown
+ // Wise words. - Bxil

```

Comment:
Is bxil the new pete? Find out in next week's exciting installment!

Reply:
![image](http://i.imgur.com/CtGToHC.gif)